### PR TITLE
install docs updated to jessie and stretch

### PR DESCRIPTION
**Pull request by boyska**
look ma, no virtualenv!

This PR only changes docs, nothing else. Therefore to test it you need to run `python build_sphinx` and look at `build/sphinx/html/index.html`. More specifically, I have changed only the "sysadmin" section.

Also, it would be great if you could test the documented procedure itself!

### CODE REVIEW

**Comment by boyska** [comment]
rebased to include #266

**Comment by boyska** [comment]
Could someone please have a look at this PR?

File: doc/source/sysadmin.rst [hunk 1]
```diff
@@ -79,11 +99,20 @@ Start elasticsearch service::
...
+    These instructions apply both to arch and debian, but should not be needed
+    on debian: after the installation, elasticsearch is enabled and started
+    automatically.
 
 
 Start libreant
 ^^^^^^^^^^^^^^
-To execute libreant::
+
+On debian stretch, the systemd unit ``python-libreant.service`` will take care
```
Comment (boyska) [inline doc/source/sysadmin.rst]: this is wrong; it actually is `libreant.service`

**Comment by g3-3k** [comment]
Can you explain why do I need to add stretch repo? As debian user I always had many troubles in using testing repo! maybe a standard user will not add testing, if you don't offer an explanation in the wiki. ;)
by the way it compiles and it's clear.

**Comment by g3-3k** [comment]
I think this is the required package. 
https://packages.debian.org/search?keywords=python-dominate

Is it possible

**Comment by boyska** [comment]
cocco->nat:
>Can you explain why do I need to add stretch repo?

yes, `python-dominate` is the reason. You're also right about your
concerns adding stretch repo to a jessie. I'll a warning providing
additional explanation and providing a command to set the default
release to jessie, so that you won't find with everything upgraded.

**Comment by g3-3k** [comment]
I installed in a Virtualenv, it works fine.
Maybe a lil explanation of what happen after installation should be offered.
something like:
Libreant has configuration file in /etc/libreant, use this folder for presets too, maybe you want to create a daemon in systemd like this:
 (e qua la unit)

**Comment by boyska** [comment]
cocco->nat:
>Maybe a lil explanation of what happen after installation should be offered.
>something like:
>Libreant has configuration file in /etc/libreant

actually it has not. for debian packaging I configured this explicitly,
but that's generally false.
The point is that I should include so many steps!
* Create a dedicated user
* Create configuration directory
* Create systemd unit (and what about non-systemd distros?)
* Provide a "suggested" `libreant.conf`
Basically everything I do in the debian package should be rewritten
here, explained, AND made portable to different systems. I think that it
is way too much for an installation guide! If we want to support a
specific additional distribution, we'd better create a package for it.
Is there a realistic case in which the sysadmin:
* can't use a supported distro
* wants to install system-wide
* can't just look at the debian package and "steal" from it
?

See also how #227 is still open and there is no consensus on the
"default" paths

>use this folder for presets too

Again, see #227 : there is no consensus on the canonical `PRESET_PATHS`,
and nothing about it is implemented in the debian package. This is also
related to #237: only one (and very minimal) preset is provided.

**Pull request closed without merging**